test/index.js

```diff
@@ -39,9 +39,9 @@ describe('emit asserts for: ', () => {
             assert.equal(trim(actual), trim(expected));
 
             // Check message output
-            const expectedMessages = fs.readFileSync(path.join(fixtureDir, 'expected.json'));
-            const actualMessages = fs.readFileSync(path.join(fixtureDir, 'actual.json'));
-            assert.equal(trim(actualMessages), trim(expectedMessages));
+            const expectedMessages = require(path.join(fixtureDir, 'expected.json'));
```

Comment:
changed this test to rely on deepEqual instead of text string content. different object key order was breaking tests